test/test.js

```diff
@@ -2416,6 +2416,37 @@ describe('Page', function() {
       await page.addStyleTag({ content: 'body { background-color: green; }' });
       expect(await page.evaluate(`window.getComputedStyle(document.querySelector('body')).getPropertyValue('background-color')`)).toBe('rgb(0, 128, 0)');
     }));
+
+    it('should returning element handle work with a url', SX(async function() {
```

Comment:
These are a lot of tests for a not-involved code.

Can you please make sure that `await page.addStyleTag` in three already-existing tests return `ElementHandle`? This would be enough for the change.

Reply:
Thanks for the review, I removed these and just check `.asElement()`. Also updated the api doc.